config.h

```diff
@@ -142,6 +141,36 @@ struct Cfg {
 		.dialog_control_id = IDC_PLAY_OVER_DESKTOP,
 	};
 
+	inline const static Definition TrailLength = {
+		.index = __COUNTER__,
+		.name = L"TrailLength",
+		.default_ = 3.0f,
+		.range = { 1.0f, 25.0f },
+		.dialog_control_id = IDC_TRAIL_LENGTH,
+	};
+
+	inline const static Definition TrailSpace = {
+		.index = __COUNTER__,
+		.name = L"TrailSpace",
+		.default_ = 10.0f,
+		.range = { 1.0f, 50.0f },
+		.dialog_control_id = IDC_TRAIL_SPACE,
+	};
+
+	// aka: kindly, calm the fuck down mate
+	inline const static Definition MaxTrailCount = {
```

Comment:
If we aren't going to let users change this (which I don't think we should), it shouldn't be a config option, it should just be a const static variable somewhere.

Reply:
The idea here was that it doesn't appear in the config but you can still edit it in the registry.